pint/unit.py

```diff
@@ -1109,6 +1110,9 @@ def _convert(self, value, src, dst, inplace=False):
         if isinstance(value, Decimal):
             factor = Decimal(str(factor))
 
+        if isinstance(value, Fraction):
+            factor = Fraction(str(factor))
```

Comment:
Doing this is more accurate but fractions will have larger numbers. Regarding the Decimal, there is no equivalent (to my knowledge) in Python 2.6